### ISSUE TITLE
rio_* crates are not actively maintained - maybe instead use oxrdf

### ISSUE THREAD

**Issue by jonassmedegaard** [manomayam/manas#70]
The maintainer of rio_api, rip_turtle and rio_xml [no longer actively maintain those packages](https://github.com/oxigraph/rio/issues/142) and has shifted to instead use component from his own [Oxigraph](https://github.com/oxigraph/oxigraph) project - e.g. [oxrdf](https://github.com/oxigraph/oxigraph/tree/main/lib/oxrdf).

Please consider switching from rio_* to ox* crates.

**Comment by damooo** [comment]
Yes. this should be done, once sophia project starts using ox* crates.
See https://github.com/pchampin/sophia_rs/issues/162
